src/components/compounds/AddComment/style.scss

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
While it is possible to select elements by ID in CSS, it should generally be considered an anti-pattern. ID selectors introduce an unnecessarily high level of specificity to your rule declarations, and they are not reusable.
see https://github.com/airbnb/css#id-selectors

Reply:
Thanks for the feedback.
However, `The high level of specificity rule declarations` is highly necessary to my achieving my target. Also, I do not intend to reuse the styles I applied to an element of which I selected with an ID. If that had not been the case, I would have gone for `classes`.